frontend/src/app/_components/navbar/Navbar.module.css

```diff
@@ -0,0 +1,48 @@
+.navigation {
+    padding: 20px 60px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    gap: 20px;
+}
+
+.container {
+    display: flex;
+    align-items: center;
+    flex-grow: 1;
+    gap: 20px;
+}
+
+.nav_links {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    font-size: 16px;
+    gap: 40px;
+}
+
+.link {
+    font-weight: 500;
+    text-decoration: none;
+    color: inherit;
```

Comment:
Why have you added color: inherit?

Reply:
This stops the links from being blue, it makes the text colour inherited from app instead. I thought better to set it as inherit so it can change based on global app styles than on the individual links :)

Reply:
I don't think it will have that behaviour. "The inherit CSS keyword causes the element to take the computed value of the property from its parent element." [https://developer.mozilla.org/en-US/docs/Web/CSS/inherit] 

If you use inherit, it will inherit the color from it's parent element not globally.

But if you want to set it in global.css that is different and you are better off just doing that.

Play around with the code and you'll see what I mean i.e. set global link style to orange and keep the rest of your code the same.

Reply:
Also using the global.css file keeps a clearer source of truth.